test.js

```diff
@@ -900,8 +872,7 @@ test('createAsyncIterator creates AsyncIterator from another AsyncIterator', ()
 })
 
 test('createAsyncIterator creates AsyncIterator for string literal', () => {
-  // $FlowFixMe expected function override to apply.
-  var iterator = createAsyncIterator('ABC')
+  var iterator = createAsyncIterator((('ABC': any): {length: number}))
```

Comment:
I'm curious why this annotation is needed. Shouldn't flow know that `string` has a `length: number` prop?